nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -1035,11 +1035,27 @@ public Nullness getGenericParameterNullnessAtInvocation(
       }
     }
 
-    if (!(tree.getMethodSelect() instanceof MemberSelectTree) || invokedMethodSymbol.isStatic()) {
+    if (tree instanceof MethodInvocationTree) {
+      if (!(((MethodInvocationTree) tree).getMethodSelect() instanceof MemberSelectTree)
+          || invokedMethodSymbol.isStatic()) {
+        return Nullness.NONNULL;
+      }
+    } else if (!(tree instanceof NewClassTree) || invokedMethodSymbol.isStatic()) {
```

Comment:
If we hit this `else` case I believe it must be true that `tree instanceof NewClassTree`.  Further, if it's a `NewClassTree`, the invoked method is not static.  So I think we should delete this entire `else if` case.